protocol/scripts/markets/get_isolated_market_insurance_fund.py

```
@@ -0,0 +1,197 @@
+import requests
+import subprocess
+import math
```

Ruff

`math` imported but unused

Remove unused import: `math`

(F401)